contracts/minting/Controller.sol

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright CENTRE SECZ 2021
```

Comment:
Should be 2018 - 2021 since it used to be 2018

Reply:
done https://github.com/centrehq/centre-tokens/pull/327/commits/7d5e39e203651601a44a157bf5fb55497ab595f4